### DISCUSSION TITLE
Possible Typos in Documentation

### DISCUSSION THREAD

**Discussion by nickthiru** [Moddable-OpenSource/moddable#997] [General]
Hi,

I think I may have picked up some typos in the documentation.

In `moddale/documentation/tools/tools.md`:

1.
![image](https://user-images.githubusercontent.com/47299208/209253608-f7e68f08-088a-41e7-aac4-77dadf483e50.png)
I think the first command should be `cd $MODDABLE/examples/piu/balls`

2.
![image](https://user-images.githubusercontent.com/47299208/209253825-f16150cd-e993-423b-93f7-928214007f4a.png)
I think think the `mcconfig` command here should actually be the `mcrun` command.

**Comment by phoddie** [comment]
Those are both definitely mistakes. Thank you for the corrections! They'll be included in the next Moddable SDK update, which is scheduled for tomorrow.

**Reply by nickthiru** [reply]
Just an update.....I noticed that the second correction has been implemented in the docs, but not the first one i.e. incorrect file path to the example.

**Reply by andycarle** [reply]
Fixed!

**Comment by nickthiru** [comment]
Ref: https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/devices/esp32.md

![image](https://user-images.githubusercontent.com/47299208/216239040-829fc777-013a-401e-b5cc-cb73efb7c758.png)

I believe the first one should read: "...data usually available in xsbug **is** output to..."

The second marking shows a duplicate word.

**Reply by andycarle** [reply]
Thank you! I'll get those fixed.

**Comment by nickthiru** [comment]
My apologies @andycarle @phoddie. I just came across the "Contributing to the Moddable SDK" doc and realized that I should be making the corrections myself and submitting PRs. I will do so going forward. Thanks.

**Reply by phoddie** [reply]
No problem. We're happy to have the reports. PRs would be even better.

**Reply by nickthiru** [reply]
> PRs would be even better.

Yup, seems like no-brainer to me now. I'm brushing up on my Git knowledge right now to make sure I do this properly i.e. create a branch, make the changes, create a PR? Sorry; a bit of noob here :-) . But this has turned out to be a great opportunity to improve my Git skills. Thanks again!
